src/api/contact/contact.resolver.ts

```diff
@@ -108,7 +108,9 @@ export class WalletContactResolver {
   @ResolveField()
   async encryption_pubkey(@Parent() parent: WalletContactParent) {
     if (!parent.money_address) return null;
-    return this.isomorphicLnurl.getPubkey(parent.money_address);
+    const pubkey = await this.isomorphicLnurl.getPubkey(parent.money_address);
+    if (pubkey === null) return null;
```

Comment:
@bufo24 should we throw an error here?
```suggestion
    if (pubkey === null) {
      throw new Error(`Pubkey not found for money address: ${parent.money_address}`);
    }
```

Reply:
we can leave this file as is.
`getPubkey` already returns either a string or null, and this field is nullable so this field resolver is all good

Reply:
Right!